traffic_ops/app/lib/API/User.pm

```diff
@@ -599,6 +599,123 @@ sub current {
 	}
 }
 
+# Designated handler for the deprecated path to updating current users
+sub user_current_update {
```

Comment:
Why did a new handler needed to be added for deprecation notice? Shouldn't we just be adding a deprecation message to the response on the current handler?

Reply:
The current handler is also the handler for `PUT /user/current`, which isn't deprecated. Since the rendering happens before the handler returns, I couldn't just wrap it, and if there's a way to access what path was requested from within a Mojolicious handler then I don't know it. So this seemed to be my only option.

Reply:
ugh! i hope that we dont have to do that with a lot of the other endpoints we want to depreciate. Cool let me pull and test this then!

Reply:
generally I don't think so.